TimeSliceNet/TimeSlice/Converters/EnforceTimeZoneOffsetConverter.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Globalization;
+using System.Text.Json;
+
+namespace TimeSlice.Converters
+{
+
+    /// <summary>
+    /// string extension methods, that are only used internally
+    /// </summary>
+    internal static class DateTimeOffsetConverterStringExtension
+    {
+        /// <summary>
+        /// similar to "O" but without the sub seconds
+        /// </summary>
+        internal const string TheOneAndOnlyDatetimeformat = "yyyy-MM-ddTHH:mm:sszzzz";
+
+        /// <summary>
+        /// converts <paramref name="dateTimeString"/> to a datetime with offset 0
+        /// </summary>
+        /// <param name="dateTimeString"></param>
+        /// <returns>null iff <paramref name="dateTimeString"/> is null</returns>
+        /// <exception cref="FormatException">iff <paramref name="dateTimeString"/> is not parseable with an offset</exception>
+        internal static DateTimeOffset? ToUtcDateTime(this string dateTimeString)
```

Comment:
Shouldn't that be called ToUtcDateTimeOffset then?

Reply:
https://github.com/Hochfrequenz/time_slice_net/pull/10/commits/eb60fdd08ecb38692e2243e8aa5e98b387029e29